runner/runner/main.py

```diff
@@ -128,14 +130,17 @@ def load_native(config: Mapping[str, Any]) -> None:
     )
     subprocess_run(
         command_lst_sbst,
-        env_override=dict(ILLIXR_DATA=str(data_path), ILLIXR_DEMO_DATA=str(demo_data_path), KIMERA_ROOT=config["loader"]["kimera_path"]),
+        env_override=dict(
+            KIMERA_ROOT=config["loader"]["kimera_path"], **consts_map_pathified,
```

Comment:
Can we factor `kimera_path` into a global module def too?

Reply:
Definitely! I wasn't sure what the desired behavior was for kimera, since the path and the plugin is/was not used in all of the yaml configurations. Should it be a default global definition?

Reply:
Actually, it's probably better to handle this a different way.

Reply:
Got it. Will discuss more about `__FILE__` then

Reply:
Did you guys file an issue for this?